mParticle-Example/MPKitExample.m

```diff
@@ -1,24 +1,24 @@
 #import "MPKitExample.h"
+#import <Pilgrim/Pilgrim.h>
 
-/* Import your header file here
-*/
-//#if defined(__has_include) && __has_include(<Example/Example.h>)
-//#import <Example/Example.h>
-//#else
-//#import "Example.h"
-//#endif
+NSString *const PILGRIM_SDK_KEY = @"pilgrim_sdk_key";
+NSString *const PILGRIM_SDK_SECRET = @"pilgrim_sdk_secret";
+NSString *const PERSIST_LOGS = @"pilgrim_sdk_persistent_logs";
+NSString *const MPARTICLE_USER_ID = @"mParticleUserId";
 
```

Comment:
don't forget to add in nullability annotations to the header and impl files:
ie. NS_ASSUME_NONNULL_BEGIN/END

Reply:
👍